### ISSUE TITLE
Increasing pixel size of glow doesn't work, frozen at 2px

### ISSUE THREAD

**Issue by RandomNerd1** [FiniteSingularity/obs-stroke-glow-shadow#50]
I can increase the number just fine, but it doesn't look like it's visually growing as I do, and when I leave the filter settings and come back, it's set back to 2px.

Am I doing something wrong?

https://i.imgur.com/cYfKtgt.png

**Comment by FiniteSingularity** [comment]
Hi @RandomNerd1,

Apologies- I just saw this issue.

Is the problem that the slider wont let you go past 2px?  Or that only two pixels are rendering, even if the slider goes larger?

**Comment by RandomNerd1** [comment]
The latter, I can move the slider, click the little arrows, or key in a larger number, no problem, but the glow will not get bigger.

**Comment by FiniteSingularity** [comment]
It sounds like you might be using a source that doesn't have enough padding to show the full glow or shadow.  Give the info in this comment a try: https://github.com/FiniteSingularity/obs-stroke-glow-shadow/issues/35#issuecomment-1865419559 and let me know if it helps.  If not, can you take some screen shots of what you're seeing (on the actual source in OBS), and we can figure out what's going on.

**Comment by RandomNerd1** [comment]
Because it's on my webcam and I didn't want to dox myself, I went to use some other source, and suddenly realized the issue had vanished. Went back to my webcam source and noticed that if I moved the slider to the extremes rapidly, I'd sometimes see a very brief flash of the large size.

Well, long story short, it was the audio move (part of Exeldro's "Move Transition" plugin) filter I had set on my mic to have the glow grow and shrink as I spoke, that was the culprit; namely, the "Base Value" value was set to 2 there, and I forgot about that, so when I changed my mind about how large I wanted to glow to start out at, I only tried changing it in the glow filter itself.

https://i.imgur.com/mxbrZzQ.png

I did the dumb. :) Closing ticket, thank you for trying to help!
